### ISSUE TITLE
Some scope for potential cleanup in the pom?

### ISSUE THREAD

**Issue by paulk-asert** [peter-gergely-horvath/dyna4jdbc#23]
Hi, quickly scanning the source code, it seemed like it was Java, so perhaps there is no compile dependency on renjin or scala? Perhaps they should be marked as optional, or if in tests as test scope?
Also, Renjin seemed to have version RELEASE in the pom which didn't seem to exist?

Anyway, nice library, I had fun playing in my little Groovy->SQL->Groovy trial:
```
@Grab('com.github.peter-gergely-horvath:dyna4jdbc:1.0.0')
@GrabExclude('org.scala-lang:scala-compiler')
@GrabExclude('org.renjin:renjin-script-engine')
@GrabExclude('org.scala-lang:scala-library')
@GrabConfig(systemClassLoader=true)
import groovy.sql.Sql

def script = '''
def fields = [t: 'Ticker', name: 'Name', l_cur: 'Close']
def base = 'http://www.google.com/finance/info?infotype=infoquoteall&q'
def url = new URL("$base=NASDAQ:AAPL,IBM,MSFT,GOOG")
def json = url.text.replaceFirst("//", "")
def data = new groovy.json.JsonSlurper().parseText(json)
println fields.values().collect{ "$it::" }.join('\t')
data.each { row -> println fields.keySet().collect{ row[it] }.join('\t') }
'''

def url = 'jdbc:dyna4jdbc:scriptengine:groovy'
Sql.withInstance(url) { sql ->
  sql.eachRow(script){ row -> println "$row.Ticker $row.Close" }
}
```
Would be nice to get rid of the @GrabExclude statements but a minor issue for now.

**Comment by peter-gergely-horvath** [comment]
This has been fixed in the latest release 1.8.0

**Comment by paulk-asert** [comment]
Thanks!
